composer.json

```diff
@@ -24,6 +24,12 @@
         "source": "https://github.com/sulu/skeleton",
         "docs": "http://docs.sulu.io/"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/acrobat/Routing.git"
```

Comment:
this can be removed when https://github.com/symfony-cmf/Routing/pull/245 is merged